backend/mora/service/manager.py

```diff
@@ -51,10 +51,15 @@ def finder(cls, c, type, objid):
                 ou = orgunit.get_one_orgunit(
                     c, objid, details=orgunit.UnitDetails.FULL
                 )
-                if not ou or mapping.PARENT not in ou:
+                if not ou:
                     return found
-                upperid = ou[mapping.PARENT][mapping.UUID]
-                found = super().finder(c, type, upperid)
+                upper = ou.get(mapping.PARENT)
+                if not upper:
+                    return found
+                objid = upper.get(mapping.UUID)
+                if not objid:
```

Comment:
So, if I'm reading this right, this is in case we found a parent, but the parent doesn't have a UUID. Has that ever happened? Can it even happen?

Reply:
I have no idea. I read a value from a dictionary and with this I check not only if there was no UUID in the parent, but also if it was None. 

Same construct as with upper where it was indeed needed as there was a parent-key but the value was None.

The way I see it, it will hopefully never be needed, but it doesn't hurt.